contrib/chart/backstage/templates/ingress.yaml

```diff
@@ -29,30 +29,59 @@ spec:
         - {{ $frontendUrl.host }}
         - {{ $backendUrl.host }}
         - {{ $lighthouseUrl.host }}
-
   rules:
     - host: {{ $frontendUrl.host }}
       http:
         paths:
           - path: /
+            {{- if ge .Capabilities.KubeVersion.Minor "19" }}
+            pathType: Prefix
+            {{- end }}
             backend:
+              {{- if lt .Capabilities.KubeVersion.Minor "19" }}
               serviceName: {{ include "frontend.serviceName" . }}
               servicePort: 80
+              {{- else if ge .Capabilities.KubeVersion.Minor "19" }}
```

Comment:
Would assume that this could just be an `{{- else }}` right?

Reply:
Yes - have fixed across the whole chart